hover/effects/icon-transitions/_icon-float-away.sass

```diff
@@ -1,3 +1,5 @@
+// This mixin requires that 2 identical icons to be present in the html
```

Comment:
`requires that 2 indentical icons to be present`:

Either
`requires that 2 indentical icons be present`
or
`requires 2 indentical icons to be present`